mirage/fixtures/registration-providers.ts

```diff
@@ -8,8 +8,13 @@ function randomAssets() {
     };
 }
 
+function brandAssets() {
+    return [server.create('brand-asset')];
+}
+
 interface MirageRegistrationProvider extends RegistrationProvider {
     licensesAcceptableIds: string[];
+    brandAssets: any;
```

Comment:
HACK!! I had to do this to fix a ts error. Not sure what the type should be here.